tests/fixtures/gpsies_example.gpx

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<gpx xmlns="http://www.topografix.com/GPX/1/1" xmlns:gpsies="https://www.gpsies.com/GPX/1/0" creator="AllTrails https://www.alltrails.com/ - Innrunde" version="1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd https://www.gpsies.com/GPX/1/0 https://www.gpsies.com/gpsies.xsd">
+  <metadata>
+    <name>Innrunde</name>
+    <link href="https://www.gpsies.com/">
+      <text>Innrunde on AllTrails</text>
+      <type>trackOnWeb</type>
+    </link>
+    <time>2019-09-11T17:08:31Z</time>
+    <extensions>
+      <gpsies:property>round trip</gpsies:property>
+      <gpsies:trackLengthMeter>6998.28996147978</gpsies:trackLengthMeter>
+      <gpsies:totalAscentMeter>91.0</gpsies:totalAscentMeter>
+      <gpsies:totalDescentMeter>91.0</gpsies:totalDescentMeter>
+      <gpsies:minHeightMeter>297.0</gpsies:minHeightMeter>
+      <gpsies:maxHeightMeter>317.0</gpsies:maxHeightMeter>
+    </extensions>
+  </metadata>
+  <trk>
+    <name>Innrunde on AllTrails</name>
+    <link href="https://www.gpsies.com/map.do">
```

Comment:
Could you add a test to make sure this link data is parsed and retrievable from `result.tracks[0]`?